README.md

```diff
@@ -112,3 +93,27 @@ spec:
         - service: sample-svc-asia1
           weight: 0
 ```
+
+## Implementation details
+
+### Failover criteria
+
+The failover criteria is readiness failures on the targeted Pods. This is
+directly reflected on the Endpoints pointing to those Pods: only when Pods are
```

Comment:
nit: wdyt about "directly reflected on the Endpoints object associated with those Pods". In my mind it would make sense that the two objects are associated rather than one pointing to the other (or vice versa).

Reply:
:+1: